code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -144,6 +144,8 @@
 				/obj/item/stamp/chameleon = 2,
 				/obj/item/clothing/shoes/chameleon/noslip = 5,
 				/obj/item/clothing/mask/chameleon = 2,
+				/obj/item/clothing/mask/gas/voice_modulator = 2,
+				/obj/item/clothing/mask/gas/voice_modulator/chameleon = 2,
```

Comment:
You need to update the line "" = 68 below this to "" = 64. You are adding 4 items to the loot table, so 4 needs to be removed from the 'empty' field.

Reply:
@datlo thanks! Fixed in the commit below.